src/Shared/NativeMethodsShared.cs

```diff
@@ -886,6 +908,18 @@ internal static DateTime GetLastWriteFileUtcTime(string fullPath)
                 WIN32_FILE_ATTRIBUTE_DATA data = new WIN32_FILE_ATTRIBUTE_DATA();
                 bool success = false;
 
+                if (!Is64BitProcess)
+                {
+                    //32 bit processes do not have access to system32. to get time for files there we must remap to a special
+                    string windowsDirectory = Environment.SystemDirectory.ToLowerInvariant();
+                    string fullPathUpper = fullPath.ToLowerInvariant();
+
+                    if (fullPathUpper.StartsWith(windowsDirectory))
```

Comment:
You can use `StartsWith(String, StringComparison)` to avoid the extra ToLower string allocations.

Reply:
ok, will change